tests/tensor_search/integ_tests/test_add_documents_unstructured.py

```diff
@@ -347,7 +350,9 @@ def test_add_documents_list_data_type_validation(self):
                     )
                 ).dict(exclude_none=True, by_alias=True)
                 assert add_res['errors'] is True
-                assert all(['error' in item for item in add_res['items'] if item['_id'].startswith('to_fail')])
+                assert all(['error' in item for item in add_res['items']])
+                assert all(['Unstructured Marqo index only supports string lists.' in item['message']
```

Comment:
Exception messages and log messages shouldn't end with a period. I think we're quite inconsistent with this pattern

Reply:
Should I change it now? Is this a breaking change?

Reply:
Need to do more research.